corehq/apps/hqadmin/service_checks.py

```diff
@@ -194,7 +194,9 @@ def check_couch():
 
 def check_formplayer():
     try:
-        res = requests.get('{}/serverup'.format(get_formplayer_url()), timeout=5)
+        # Setting verify=False in this request keeps this from failing for urls with unsigned certificates.
+        # Allowing this because the certificate will always be unsigned for the fullstack deploy.
```

Comment:
Nit: I'd say
> Setting verify=False in this request keeps this from failing for urls with self-signed certificates.
> Allowing this because the certificate will always be self-signed in the "provable deploy" bootstrapping test in commcare-cloud

Reply:
https://github.com/dimagi/commcare-hq/pull/21483/commits/37553b9156db96559b8be47df6b8ef9e183399b7